tasks/page/makeReadme.js

```diff
@@ -10,40 +10,49 @@ const converter = new htmlToJsx({createClass: true, outputClassName: 'Readme'})
 
 if (require.main !== module) {
   module.exports = function(config, folder) {
+    const folders = folder.split('/')
     const pattern = `${folder}/readme?(\.markdown|\.mdown|\.mkdn|\.md|\.rst)`
     const readmes = glob.sync(pattern, {nocase: true})
     const deps = [`build/.temp/${folder}/info.json`]
     if (readmes.length > 0) {
       deps.push(readmes[0])
     }
+
     const targets = [`build/.temp/${folder}/readme.jsx`]
+
+    if (folders.length > 4) {
+      const projectPath = folders.splice(4).join('/')
+      targets.push(projectPath)
+    }
+
```

Comment:
Is this the best way to detect `multi` repos? We should at least explain it in a comment.

Reply:
Added a comment for now, but I think it can be done better. 

I have used this same logic in a few others places, after this gets merged, I can make another PR changing all instances to a better approach.

Reply:
I think the comment and the variable naming could be a bit clearer, mentioning `multi` explicitly.